backend/organization/views/project_views.py

```diff
@@ -168,6 +170,38 @@ def post(self, request, *args, **kwargs):
         # Call the standard list method which handles filtering and pagination
         return self.list(request, *args, **kwargs)
 
+    def get(self, request, *args, **kwargs):
```

Comment:
Why are you using this get method instead of get_queryset? You seem to be manually implementing most of the native get_queryset behavior here again? :thinking:

Reply:
I assume this is mainly to use a custom serializer context if I understand the function correctly?
Could we instead use `get_serializer_context`for this?

Reply:
Revertet it and added a get_serializer_context function, so this is done :heavy_check_mark: